test/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQSamples.cs

```diff
@@ -65,18 +59,17 @@ public static class RabbitMQSamples
         // So you can add items to the queue while the sample is running, and the trigger will be called until the queue is empty.
         public static void QueueTrigger_RabbitMQOutput(
             [QueueTrigger(@"samples-rabbitmq-messages")] TestClass message,
-            [RabbitMQ(
-                HostName = "localhost",
-                QueueName = "queue")] out TestClass outputMessage,
+            [RabbitMQ(QueueName = "queue")] out TestClass outputMessage,
             ILogger logger)
         {
             outputMessage = message;
             logger.LogInformation($"RabbitMQ output binding message: {JsonConvert.SerializeObject(outputMessage)}");
         }
 
         // Trigger samples
+        // Defaults to localhost if HostName is not specified

```

Comment:
Defaults to localhost if HostName is not specified and connection string is not set in  appsettings.json